code/obj/machinery/photocopier.dm

```diff
@@ -29,246 +39,819 @@ TYPEINFO(/obj/machinery/photocopier)
+		else
+			src.icon_state = "scan_id"
+		boutput(user, "You put the card on the scan bed, close the lid, and press start...")
+		effects_scanning(I)
+		src.print_info["name"] = I.name
+		src.print_info["icon_state"] = I.icon_state
+		src.print_info["item_state"] = I.item_state
+		src.print_info["desc"] = I.desc
+		src.print_info["pronouns"] = I.pronouns
+		src.print_info["access"] = I.access.Copy()
+		src.print_info["registered"] = I.registered
+		src.print_info["assignment"] = I.assignment
+		src.print_info["title"] = I.title
+		src.print_info["reagent_account"] = I.reagent_account
+		src.print_info["keep_icon"] = I.keep_icon
+		src.print_info["money"] = I.money
+		src.print_info["pin"] = I.pin
+		src.print_info["cardfile"] = I.cardfile
+		src.print_type = "id"
+	proc/scan_poster(var/obj/item/poster/titled_photo/W, var/mob/user)
+		scan_setup(W, user)
+		src.icon_state = "papper"
+		boutput(user, "You put the poster on the scan bed, close the lid, and press start...")
+		effects_scanning(W)
+		src.print_info["name"] = W.name
+		src.print_info["desc"] = W.desc
+		src.print_info["icon_state"] = W.icon_state
+		src.print_info["poster_image"] = W.poster_image
+		src.print_info["poster_image_old"] = W.poster_image_old
+		src.print_info["photo"] = W.photo
+		src.print_info["line_title"] = W.line_title
+		src.print_info["poster_HTML"] = W.poster_HTML
+		src.print_info["line_photo_subtitle"] = W.line_photo_subtitle
+		src.print_info["line_below_photo"] = W.line_below_photo
+		src.print_info["line_b1"] = W.line_b1
+		src.print_info["line_b2"] = W.line_b2
+		src.print_info["line_b3"] = W.line_b3
+		src.print_info["author"] = W.author
+		src.print_info["plist"] = W.plist.Copy()
+		src.print_type = "poster_wanted"
+	proc/scan_reagent_scanner(var/obj/item/device/reagentscanner/R, var/mob/user)
+		src.reset_all()
+		boutput(user, "You upload the reagent scanner's previous results to the photocopier.")
+		src.print_info["name"] = "Reagent Scanner Results"
+		src.print_info["desc"] = "List of past results from a reagent scanner."
+		var/scan_data = "<li><h3>Reagent Scanner Results</h3></li><hr>" + R.scan_results
+		src.print_info["info"] = scan_data
+		src.print_info["stamps"] = null
+		src.print_info["form_fields"] = list()
+		src.print_info["field_counter"] = 1
+		src.print_info["icon_state"] = "paper_blank"
+		src.print_info["sizex"] = 0
+		src.print_info["sizey"] = 0
+		src.print_info["scrollbar"] = TRUE
+		src.print_info["overlays"] = list()
+		src.print_type = "paper"
+	proc/scan_bible(var/obj/item/bible/B, var/mob/user)
+		scan_setup(B, user)
+		boutput(user, "You open the Bible, press a passage onto the scan bed, and press start...")
+		src.icon_state = "scan_book"
+		effects_scanning(B)
+		src.print_info["name"] = "Matthew 25:14-30"
+		src.print_info["desc"] = "Photocopy of a Bible passage."
+		// Source: https://www.vatican.va/archive/ENG0839/__PVY.HTM
+		src.print_info["info"] = file2text("strings/books/bible_matthew25_14_30.txt")
```

Comment:
yeah, no. not stirring up this issue. nix this entirely, please.

_**Do Not Merge until this is removed**_

Reply:
> yeah, no. not stirring up this issue. nix this entirely, please.

Oh, okay? I didn't think it was any more of an issue than the Bible already being in the game. I mean what else would the Bible do, aside from smiting you of course.